make it work in locales where [a-z] DOES NOT DEFINE WHOLE ALPHABET, like et_EE locale

--- ./rc.d/init.d/network.america-rules	2005-07-07 19:28:21.000000000 +0300
+++ ./rc.d/init.d/network	2005-07-07 19:27:25.000000000 +0300
@@ -134,7 +134,7 @@
 		. /etc/rc.d/init.d/functions; \
 		cd /etc/sysconfig/interfaces && ls -1 ifcfg* | \
 		egrep -v '(ifcfg-lo|ifcfg-sit|ifcfg-atm|ifcfg-lec|ifcfg-nas|ifcfg-br|ifcfg-(.*)\.(.*))' | \
-		egrep 'ifcfg-[a-z0-9\.:]+$' | \
+		LC_ALL=C egrep 'ifcfg-[a-z0-9\.:]+$' | \
 		for i in `cat`; do \
 		    ONBOOT=""; . /etc/sysconfig/interfaces/"$i"; is_yes "$ONBOOT" && echo "$i"; \
 		done | \
@@ -143,7 +143,7 @@
 		. /etc/rc.d/init.d/functions; \
 		cd /etc/sysconfig/interfaces && ls -1 ifcfg* | \
 		egrep 'ifcfg-(.*)\.(.*)' | \
-		egrep 'ifcfg-[a-z0-9\.]+$' | \
+		LC_ALL=C egrep 'ifcfg-[a-z0-9\.]+$' | \
 		for i in `cat`; do \
 		    ONBOOT=""; . /etc/sysconfig/interfaces/"$i"; is_yes "$ONBOOT" && echo "$i"; \
 		done | \
@@ -151,7 +151,7 @@
 	interfaces_br_boot=$((
 		. /etc/rc.d/init.d/functions; \
 		cd /etc/sysconfig/interfaces && ls -1 ifcfg-br* | \
-		egrep 'ifcfg-[a-z0-9\.]+$' | \
+		LC_ALL=C egrep 'ifcfg-[a-z0-9\.]+$' | \
 		for i in `cat`; do \
 		    ONBOOT=""; . /etc/sysconfig/interfaces/"$i"; is_yes "$ONBOOT" && echo "$i"; \
 		done | \
@@ -159,7 +159,7 @@
 	interfaces_sit_boot=$((
 		. /etc/rc.d/init.d/functions; \
 		cd /etc/sysconfig/interfaces && ls -1 ifcfg-sit* | \
-		egrep 'ifcfg-[a-z0-9]+$' | \
+		LC_ALL=C egrep 'ifcfg-[a-z0-9]+$' | \
 		for i in `cat`; do \
 		    ONBOOT=""; . /etc/sysconfig/interfaces/"$i"; is_yes "$ONBOOT" && echo "$i"; \
 		done | \
